src/main/java/roomescape/member/MemberService.java

```diff
@@ -7,15 +7,24 @@
 @Transactional(readOnly = true)
 public class MemberService {
 
-    private final MemberDao memberDao;
+    private final MemberRepository memberRepo;
 
-    public MemberService(MemberDao memberDao) {
-        this.memberDao = memberDao;
+    public MemberService(MemberRepository memberRepo) {
+        this.memberRepo = memberRepo;
     }
 
     @Transactional
     public MemberResponse createMember(MemberRequest memberRequest) {
-        Member member = memberDao.save(new Member(memberRequest.getName(), memberRequest.getEmail(), memberRequest.getPassword(), "USER"));
+        memberRepo.findByEmail(memberRequest.getEmail())
+                .ifPresent(m -> { throw new IllegalArgumentException("존재하는 이메일입니다."); });
```

Comment:
- 요건 중요하지 않은 코멘트입니다

`CrudRepository`에서 exist라는 키워드로 메서드 쿼리를 만들 수 있어요!
```suggestion
        if (memberRepo.existsByEmail(memberRequest.getEmail())) {
            throw new IllegalArgumentException("존재하는 이메일입니다.");
        }
```